860_A1_Q4.ipynb

```diff
@@ -132,11 +154,53 @@
     "## create object of Linearregression class\n",
     "reg = lp()\n",
     "## Fit the linear rgerssion\n",
-    "reg = reg.fit(X_train,Y_train)\n",
     "X_train = X[:26, :]\n",
     "Y_train = Y[:26]\n",
+    "\n",
+    "reg = reg.fit(X,Y)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "26"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "Y_train.size # note Python start indexing with 0 instead of 1, could use X[:25,:] and Y[:25] instead"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Model Function ->> Y_Prediction = -39.5389 + 1.6693*Height + 101.7732*Experience\n",
+      "The mean sqaured error with hieght and experince as our model feature is : 3092.910\n",
+      "The R-Squared value is: 0.83\n"
+     ]
+    }
+   ],
+   "source": [
+    "X_train = X[:25, :]\n",
+    "Y_train = Y[:25]\n",
+    "\n",
     "reg = reg.fit(X_train,Y_train)\n",
-    "Y_pred = reg.predict(X).astype('int64')\n",
+    "# Y_pred = reg.predict(X).astype('int64') # [Paul: not sure why you would put all Y_pred into integers]\n",
```

Comment:
I was getting some mismatch error in calculation of R square, So I changed the format of Y_pred to match with Y. #